manager/workload/filter.go

```diff
@@ -0,0 +1,26 @@
+package workload
+
+import (
+	"fmt"
+
+	"github.com/projecteru2/agent/types"
+	"github.com/projecteru2/agent/utils"
+	"github.com/projecteru2/core/cluster"
+)
+
+func (m *Manager) getFilter(extend map[string]string) []types.KV {
+	var f []types.KV
+	f = append(f, types.KV{Key: "label", Value: fmt.Sprintf("%s=1", cluster.ERUMark)})
+
+	if m.config.CheckOnlyMine && utils.UseLabelAsFilter() {
+		f = append(f, types.KV{Key: "label", Value: fmt.Sprintf("eru.nodename=%s", m.config.HostName)})
+		if m.coreIdentifier != "" {
```

Comment:
dont think here is ```coreIdentifier```,  here is identify which ``` store ``` used, core is just one of implementation.